### ISSUE TITLE
Learning rate for training retinanet with the parameters(imgs_per_gpu=2, num_gpus=2)

### ISSUE THREAD

**Issue by chengcchn** [open-mmlab/mmdetection#798]
In the config file "retinanet_r50_fpn_1x.py", the learning rate is 0.01, not 0.02? So, should I modify the learning rate to 0.01/8 if I use the parameters about "imgs_per_gpu=2, num_gpus=2"? Looking forward to get a solution. Thanks!

**Comment by hellock** [comment]
Yes RetinaNet and other models have different lr, which are copied from the official release.

**Comment by chengcchn** [comment]
@hellock 
Thank you for your reply! I found something interesting. When I set the learning rate as 0.02/4, I got a good result mAP 35.9, with the parameters "imgs_per_gpu=2, num_gpus=2". But the original learning rate is 0.01 for retinanet, so I should have set the learning rate as 0.01/4. I will find some time to do the experiment with the learning rate as 0.01/4.
![Snipaste_2019-06-14_19-08-35](https://user-images.githubusercontent.com/38886481/59505378-d9851080-8ed7-11e9-938f-0737e729a741.png)
